Tools/machines/perlmutter-nersc/install_gpu_dependencies.sh

```diff
@@ -109,6 +109,45 @@ CXX=$(which CC) CXXFLAGS="-DLAPACK_FORTRAN_ADD_" cmake -S $HOME/src/lapackpp -B
 cmake --build ${build_dir}/lapackpp-pm-gpu-build --target install --parallel 16
 rm -rf ${build_dir}/lapackpp-pm-gpu-build
 
+# heFFTe
+if [ -d $HOME/src/heffte ]
+then
+  cd $HOME/src/heffte
+  git fetch --prune
+  git checkout v2.4.0
+  git pull
```

Comment:
```suggestion
  git pull origin master
```
Git pull wasn't successful for me, I encountered this error: 
```
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>
```

Reply:
```suggestion
```